### PR TITLE
Run the SDK performance tests daily.

### PR DESCRIPTION
**Pull request by ghost**
Run the SDK performance tests daily.

@jorive 

Related:
https://github.com/dotnet/sdk/pull/2558

### CODE REVIEW

File: perf.groovy [hunk 1]
```diff
@@ -58,6 +58,7 @@ def static getBuildJobName(def configuration, def os) {
             archiveSettings.setAlwaysArchive()
             Utilities.addArchival(newJob, archiveSettings)
             Utilities.standardJobSetup(newJob, project, isPR, "*/${branch}")
+            Utilities.addPeriodicTrigger(newJob, "@daily", true)
```
[the remark this answers is not in the document]
Comment (johnbeisner) [reply perf.groovy]: Do you mean adding the line to the "if (isPR)" like so?

            if (isPR) {
                TriggerBuilder builder = TriggerBuilder.triggerOnPullRequest()
                builder.setGithubContext("${os} ${arch} SDK Perf Tests")
                builder.triggerOnlyOnComment()
                //Phrase is "test Windows_NT x64 SDK Perf Tests"
                builder.setCustomTriggerPhrase("(?i).*test\\W+${os}\\W+${arch}\\W+sdk\\W+perf\\W+tests.*")
                builder.triggerForBranch(branch)
                builder.emitTrigger(newJob)
            }
            else {
                TriggerBuilder builder = TriggerBuilder.triggerOnCommit()
                builder.emitTrigger(newJob)
                Utilities.addPeriodicTrigger(newJob, "@daily", true)
            }

[the remark this answers is not in the document]
Comment (mmitche) [reply perf.groovy]: Yeah

[the remark this answers is not in the document]
Comment (johnbeisner) [reply perf.groovy]: Done...

File: perf.groovy [hunk 2]
```diff
@@ -132,6 +133,7 @@ def static getBuildJobName(def configuration, def os) {
             }
 
             Utilities.standardJobSetup(newJob, project, isPR, "*/${branch}")
+            Utilities.addPeriodicTrigger(newJob, "@daily", true)
```
[the remark this answers is not in the document]
Comment (johnbeisner) [reply perf.groovy]: Done...

**Review by dsplaisted — APPROVED** [review]

**Comment by johnbeisner** [comment]
@dotnet-bot help

**Pull request merged**